src/components/App/App.scss

```diff
@@ -0,0 +1,10 @@
+.App > main {
+  color: white;
+  padding: 40px;
+}
+
+.search-line {
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-between;
+}
```

Comment:
These properties should be in another file. I don't see any class "search-line" in the App.tsx file